core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt

```diff
@@ -217,13 +247,16 @@ fun LpFileButton(
         verticalAlignment = Alignment.CenterVertically,
         modifier = modifier
             .border(
-                border = BorderStroke(width = 1.dp, MaterialTheme.colorScheme.onPrimary),
-                shape = RoundedCornerShape(4.dp)
+                border = BorderStroke(
+                    width = LeasePertTheme.sizes.stroke,
+                    MaterialTheme.colorScheme.onPrimary
```

Comment:
Please add the name of the property for the color.

Reply:
Working on the comments

Reply:
Comment resolved